include/Spectra/LinAlg/Orthogonalization.h

```diff
@@ -92,6 +110,12 @@ void partial_orthogonalisation(Matrix& in_output, Eigen::Index leftColsToSkip)
     in_output.rightCols(rightColToOrtho).colwise().normalize();
```

Comment:
The final normalization should be unnecessary, since it will be done by `QR_orthogonalisation()`. (Note that `partial_orthogonalisation()` is only used in `JensWehner_orthogonalisation()`)

Reply:
yes, but it might be used somewhere else, so I do not want to violate what the function name promises. I also do not think it will be performance critical.